values.yaml

```diff
@@ -629,3 +629,40 @@ ui:
   # This can either be YAML or a YAML-formatted multi-line templated string map
   # of the annotations to apply to the ui service
   annotations: {}
+
+# secrets-store-csi-driver-provider-vault
+csi:
+  # True if you want to install a secrets-store-csi-driver-provider-vault daemonset.
+  #
+  # Requires installing the secrets-store-csi-driver separately, see:
+  # https://github.com/kubernetes-sigs/secrets-store-csi-driver#install-the-secrets-store-csi-driver
+  #
+  # With the driver and provider installed, you can mount Vault secrets into volumes
+  # similar to the Vault Agent injector, and you can also sync those secrets into
+  # Kubernetes secrets.
+  enabled: false
+
```

Comment:
We should add an `externalVaultAddr` configurable like we have in the injector: https://www.vaultproject.io/docs/platform/k8s/helm/examples/external. Vault isn't always running on K8s (like Vault HCP) so we want to override any kind of templating configuration we do by default.

Reply:
Weirdly, there is no configuration of the vault address in the provider - that all happens in the SecretProviderClass, so nothing to do here yet. But we probably will change that, at which point we'll want to add `externalVaultAddr` as you suggest.